File: specs/master/indices/alias_management/alias/bulk_alias_request.ts

/**namespace:Indices.AliasManagement.Alias */
interface bulk_alias_request extends request {
	actions: alias_action[];
	/**ambiguous_origin*/
	Timeout: time;
	/**ambiguous_origin*/
	MasterTimeout: time;
	/**ambiguous_origin*/
	Source: string;
	/**ambiguous_origin*/
	FilterPath: string;
}

File: output/csharp/Indices/AliasManagement/Alias/BulkAliasRequest.cs
using Nest.CommonOptions;
using Nest.Indices;
using Nest.CommonAbstractions;
using System;
using System.Collections.Generic;
using System.Runtime.Serialization;
namespace Nest.Indices {

	public class BulkAliasRequest : RequestBase {
		
		[DataMember(Name="master_timeout")]
		public Time MasterTimeout { get; set; }


		[DataMember(Name="timeout")]
		public Time Timeout { get; set; }


		[DataMember(Name="actions")]
		public List<AliasAction> Actions { get; set; }

	}
}
